src/ser/mod.rs

```diff
@@ -466,25 +484,25 @@ impl<'a> ser::SerializeTuple for &'a mut Serializer {
         value.serialize(&mut **self)?;
         self.output += ",";
 
-        if let Some((ref config, _)) = self.pretty {
-            self.output += if self.separate_tuple_members() {
-                &config.new_line
-            } else {
-                " "
-            };
+        if let Some((ref config, ref pretty)) = self.pretty {
+            if pretty.indent < config.depth_limit {
+                self.output += if self.separate_tuple_members() {
+                    &config.new_line
+                } else {
+                    " "
```

Comment:
I assume this gets fixed in the follow-up? Otherwise we don't have whitespaces between tuple fields.